### PR TITLE
Update LLM chatbot with Qwen1.5 and Baichuan2

### PR DESCRIPTION
**Pull request by OpenVINO-dev-contest**
1. replace Qwen with Qwen1.5 (transformers>=4.37.0)
2. add support for Baichuan2

### CODE REVIEW

File: notebooks/254-llm-chatbot/ov_llm_model.py [hunk 1]
```diff
@@ -361,9 +361,81 @@ def _from_pretrained(
...
+        init_cls = OVQWENModel
+
+        return init_cls(
+            model=model, config=config, model_save_dir=model_cache_path.parent, **kwargs
+        )
 
 model_classes = {
     "mpt": OVMPTModel,
-    "qwen": OVQWENModel,
+    "qwen1.5": OVQWENModel,
```
Comment (eaidova) [inline notebooks/254-llm-chatbot/ov_llm_model.py:438]: there is no need in model class, qwen1.5 works with default OVModelForCausalLM

File: notebooks/254-llm-chatbot/ov_llm_model.py [hunk 2]
```diff
@@ -361,9 +361,81 @@ def _from_pretrained(
...
+            revision=revision,
+            force_download=force_download,
+            cache_dir=cache_dir,
+            file_name=file_name,
+            subfolder=subfolder,
+            local_files_only=local_files_only,
+        )
+
+        model = cls.load_model(model_cache_path, load_in_8bit=load_in_8bit)
+        init_cls = OVQWENModel
```
Comment (eaidova) [inline notebooks/254-llm-chatbot/ov_llm_model.py:430]: wrong class assigned

File: notebooks/254-llm-chatbot/ov_llm_model.py (hunk 1 above)
Comment (OpenVINO-dev-contest) [reply notebooks/254-llm-chatbot/ov_llm_model.py]: done

File: notebooks/254-llm-chatbot/ov_llm_model.py (hunk 2 above)
Comment (OpenVINO-dev-contest) [reply notebooks/254-llm-chatbot/ov_llm_model.py]: done

File: notebooks/254-llm-chatbot/254-llm-chatbot.ipynb [hunk 3]
```diff
@@ -102,7 +102,8 @@
...
+    "* **llama-2-7b-chat** - LLama 2 is the second generation of LLama models developed by Meta. Llama 2 is a collection of pre-trained and fine-tuned generative text models ranging in scale from 7 billion to 70 billion parameters. llama-2-7b-chat is 7 billions parameters version of LLama 2 finetuned and optimized for dialogue use case. More details about model can be found in the [paper](https://ai.meta.com/research/publications/llama-2-open-foundation-and-fine-tuned-chat-models/), [repository](https://github.com/facebookresearch/llama) and [HuggingFace model card](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf).\n",
+    "* **qwen1.5-7b-chat** - Qwen1.5 is the beta version of Qwen2, a transformer-based decoder-only language model pretrained on a large amount of data. Qwen1.5 is a language model series including decoder language models of different model sizes. It is based on the Transformer architecture with SwiGLU activation, attention QKV bias, group query attention, mixture of sliding window attention and full attention. You can find more details about model in the [model card](https://huggingface.co/Qwen/Qwen1.5-7B-Chat).\n",
```
Comment (eaidova) [inline notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:106]: maybe good to add also some smaller qwen1.5 versions? 1.8b for example or 0.5b?

**Review by eaidova — APPROVED** [review]

File: notebooks/254-llm-chatbot/254-llm-chatbot.ipynb (hunk 3 above)
Comment (OpenVINO-dev-contest) [reply notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:106]: 2 reason for 7b model:
1. 7b is the most trending model in Qwen's HF repo
2. Qwen team has issues on 7b model exporting, so this repo can provide some hints to their validation 

But I think our user can easily export a 1.8 or 0.5 version by replacing model id.

**Pull request merged**